contrib/building.md

```diff
@@ -181,7 +183,8 @@ del assets
 mklink /j assets "../../../../../data"
 ```
 
-Note: If you have cloned the repo previously make sure you remove your `node_modules` directory since it may contain stale dependencies which may cause the build to fail.
+> [!NOTE]
+> If you have cloned the repo previously make sure you remove your `node_modules` directory since it may contain stale dependencies which may cause the build to fail.
```

Comment:
```suggestion
> If you have cloned the repo previously make sure you remove your `node_modules` directory, as well as any `node_modules` directory of any sub-directory, since it may contain stale dependencies which may cause the build to fail.
```